test/plugins/marker.test.ts

```diff
@@ -170,4 +173,22 @@ describe('MarkerPlugin', (): void => {
             done();
         });
     });
+
+    it('!escaped marker', (done): void => {
+        const commit = new Commit('b816518030dace1b91838ae0abd56fa88eba19f5', {
+            timestamp: 0,
+            header: 'feat(Jest): subject',
+            body: '!escaped(Jest markers test)',
```

Comment:
!escaped marker marker contains no value and cannot be used as `!escaped(VALUE)`